src/domain/products/edit.tsx

```diff
@@ -60,39 +68,64 @@ const EditProductPage = ({ id }) => {
       onSubmit={onSubmit}
     >
       <ProductForm product={product} isEdit />
```

Comment:
Also, I noticed a flash of old values on submit. A hacky fix for this could be that we temporarily, while submitting, send form data to `ProductForm` instead of a product from the hook. e.g.
`<ProductForm product={submitting ? formValuesToUpdateProductMapper(formData) : product} isEdit />`

Reply:
I've also noticed it, and actually it's a recurring issue with all of our forms. It's just less noticeable with the old design as you would scroll all the way down to the page to save and not see most of the fields